server/models/Submission.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const submissionSchema = new mongoose.Schema({
+    contestID: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "contest",
+        required: true
+    },
+    userID: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'user',
+        required: true
+    },
+    active: {
+        type: Boolean,
+        default: true
+    },
+    created: {
```

Comment:
You can use the mongoose timestamps option (set to true) which will automatically generate these for you :)